core/plugins/org.fusesource.ide.camel.model.service.impl/.classpath

```diff
@@ -7,16 +7,16 @@
 	</classpathentry>
 	<classpathentry exported="true" kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-blueprint-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-maven-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-provider-karaf-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-provider-springboot-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-core-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/camel-spring-2.19.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/log4j-over-slf4j-1.7.21.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/slf4j-api-1.7.21.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/slf4j-simple-1.7.21.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-blueprint-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-maven-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-provider-karaf-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-provider-springboot-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-core-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-spring-2.20.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/log4j-over-slf4j-1.7.22.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/slf4j-api-1.7.22.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/slf4j-simple-1.7.22.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/groovy-2.4.10.jar"/>
```

Comment:
that should be groovy 2.4.12 ... you should have an error in your IDE which tells you that

Reply:
right.
I didn't have the error in the IDE because unfortunately cleaning old libs requires to restart Eclipse due locks from Java classpaths on Windows so i don't do it often.